debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlTaskContext.java

```diff
@@ -47,8 +48,38 @@ public MySqlTaskContext(Configuration config) {
         // Set up the GTID filter ...
         String gtidSetIncludes = config.getString(MySqlConnectorConfig.GTID_SOURCE_INCLUDES);
         String gtidSetExcludes = config.getString(MySqlConnectorConfig.GTID_SOURCE_EXCLUDES);
-        this.gtidSourceFilter = gtidSetIncludes != null ? Predicates.includes(gtidSetIncludes)
-                : (gtidSetExcludes != null ? Predicates.excludes(gtidSetExcludes) : null);
+        Boolean filterIncludesRegex = false;
```

Comment:
I've submitted a different PR (#183) that improves the `Predicates` class to add support for UUID-based predicates. The benefit of my approach is that it handles any combination of UUID literal and regular expressions in the GTID source filter configuration properties, but it will return the most efficient predicate implementation (e.g., using a single set for all UUID literals, and only using that one set if no regular expressions are involved). And, it includes some unit tests.

If you concur, I can merge #183 and then you can rebase your branch on `master` and simplify your PR to only deal with using the filter and defining the new config property.